src/components/MenuDropdownItem/menu-dropdown-item.css

```diff
@@ -46,17 +46,24 @@ text-overflow: ellipsis;
   flex-direction: column;
 }
 
-.card-header h5 {
+h5.wrapped {
   margin: 0 0 5px 0;
   font-size: 16px;
   color: #000;
-  white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
   cursor: pointer;
 }
 
+h5.expanded {
+  margin: 0 0 5px 0;
```

Comment:
A lot of these styles are repeated between `h5.wrapped` and `h5.expanded`. I would say it would be cleaner to have a designated `.card-header h5` style that encompasses everything but `white-space: nowrap;`, which is designated specifically for `h5.wrapped`. `h5.expanded` doesn't appear to include any unique styles.

Reply:
Good idea. I'm going to go ahead with this PR now, and let's take a look at CSS clean-up in general very soon.